locustfiles/prime_workflow.py

```diff
@@ -22,7 +23,7 @@ class PrimeWorkflowUser(MilMoveHostMixin, HttpUser):
 
     wait_time = between(0.25, 9)  # the time period to wait in between tasks (in seconds, accepts decimals and 0)
     weight = 1
-    parser = {PRIME_API_KEY: prime_api, SUPPORT_API_KEY: support_api}
+    parser = {PRIME_API_KEY: prime_api, SUPPORT_API_KEY: support_api, INTERNAL_API_KEY: internal_api}
```

Comment:
Because the prime workflow calls the prime tasks, it needs the internal api as well now that we're creating customer seed moves.